src/types.ts

```diff
@@ -100,6 +100,14 @@ export type BaseToastOptions = {
    */
   from?: "top" | "bottom";
   layout?: Layout;
+  /**
+   * Adds an action button to the toast: https://sonner.emilkowal.ski/toast#action
+   * @platform web
+   */
+  action: {
```

Comment:
Oh also remember to make it optional. It would also be better to infer this type from sonner but it’s fine for now.

Reply:
@nandorojo - I just put in a commit that extends BaseToastOptions with Sonner's ToastT but realized there are a lot of fields that BurntModule.web is not passing along yet (for example, invert, onAutoClose). I can add them all in to BurntModule.web if that's the direction you want to go, or I can revert it back to just typing `action` into the original type. Let me know which direction you prefer!

Reply:
hey @nandorojo just wanted to follow up here. Lmk which direction you want to go here!

Reply:
hey @nandorojo bumping again! Let me know how you want to go from here